pyhms/stop_conditions/gsc.py

```diff
@@ -109,7 +109,7 @@ def __init__(self, limit: int) -> None:
         self.limit = limit
 
     def __call__(self, tree: "DemeTree") -> bool:
-        problem = tree.root._problem
+        problem = tree.root._raw_problem
```

Comment:
Why is _raw_problem used here instead of _problem if _problem is the one counting the number of evaluations?

Reply:
Because _problem is counting evaluations only of the RootDeme.